tools/events/decode.go

```diff
@@ -0,0 +1,72 @@
+package event_tools
+
+import (
+	"fmt"
+	"github.com/filecoin-project/lotus/chain/types"
+
+	"regexp"
+
+	"github.com/filecoin-project/go-state-types/big"
+	"github.com/ipfs/go-cid"
+	"github.com/ipld/go-ipld-prime"
+	"github.com/ipld/go-ipld-prime/codec/dagcbor"
+	"github.com/ipld/go-ipld-prime/datamodel"
+)
+
+var (
+	cidRegex    = regexp.MustCompile("cid")
+	bigintRegex = regexp.MustCompile("balance")
```

Comment:
Can we add docs here to point to the source where we find these references for cid and/or bigints?